fedora

```diff
@@ -1,26 +1,11 @@
 ARG TAG=latest
 FROM registry.fedoraproject.org/fedora:${TAG}
 
-ARG INTEL
-
 RUN ( dnf -y update || dnf -y update ) && \
     dnf -y install \
-      make cmake git gcc-c++ gcc-gfortran ccache vim-minimal clang llvm compiler-rt sudo clang-tools-extra ninja-build libomp-devel wget zstd hwloc-devel flang hpx-devel && \
+      make cmake git gcc-c++ gcc-gfortran ccache vim-minimal clang llvm compiler-rt sudo clang-tools-extra ninja-build libomp-devel wget zstd hwloc-devel flang hpx-devel clang-tools-extra && \
```

Comment:
Ordering alphabetically
```suggestion
      make ccache clang clang-tools-extra cmake compiler-rt flang gcc-c++ gcc-gfortran git hpx-devel hwloc-devel libomp-devel llvm ninja-build sudo vim-minimal wget zstd && \
```
makes it more obvious that `clang-tools-extra` is already there.

Reply:
Good catch!